mail.pohjoistech.fi.email.json

```diff
@@ -0,0 +1,57 @@
+{
+  "providerId": "mail.pohjoistech.fi",
+  "providerName": "PohjoisMail",
+  "serviceId": "email",
+  "serviceName": "Mail",
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.pohjoistech.fi",
+  "syncRedirectDomain": "mail.pohjoistech.fi",
+  "logoUrl": "https://r2.pohjoistech.fi/logo.png",
+  "description": "Enables a domain to work with PohjoisMail, a mail service offered by Pohjoistech.",
+  "variableDescription": "ip is the IP address of the server, and key is the DKIM public key.",
+  "records": [
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "pohjoismail",
+      "essential": "OnApply",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "pohjoismail.%domain%",
+      "type": "CNAME",
+      "host": "autodiscover",
+      "essential": "OnApply",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "mailsrv.pohjoistech.fi",
+      "type": "MX",
+      "host": "@",
+      "essential": "OnApply",
+      "ttl": 14400,
+      "priority": 10
+    },
+    {
+      "spfRules": "a mx ip4:193.108.130.22 include:relay.mailchannels.net ip6:2605:6400:3:fed5:1000:201:0:2",
+      "type": "SPFM",
+      "host": "@",
+      "essential": "OnApply"
+    },
+    {
+      "type": "TXT",
+      "host": "_dmarc",
+      "data": "v=DMARC1; p=quarantine; rua=mailto:dmarc@%domain%; ruf=mailto:dmarc-forensics@%domain%; sp=reject; fo=1; adkim=s; aspf=s",
+      "essential": "OnApply",
+      "txtConflictMatchingMode": "All",
+      "ttl": 14400
+    },
+    {
+      "type": "TXT",
+      "host": "x._domainkey",
```

Comment:
`x` was just an example. Sure you want to use it as a key?

Reply:
Sorry I'm not the most familiar with DKIM. So can I just put anything on the place of the `x`?

Reply:
The spec may help: https://datatracker.ietf.org/doc/html/rfc6376#autoid-36

Anyway - the subdomain must match the signing done by the outgoing mailserver. SaaS solutions usually publish this information, otherwise consult mail server admin.

Reply:
So apparently it is supposed to be `x._domainkey`.